docs/storefront/graphql/index.mdx

```diff
@@ -591,7 +591,7 @@ Use normal GraphQL Storefront API tokens. According to the [Principle of least p
 
 ### I want to run requests from a server, and I need to support customer login
 
-Use a Customer Impersonation token and store it securely on your server like other secrets. When you need to run requests in the context of a particular customer (for example, if they've logged in to your application), send their BigCommerce Customer ID along with the request as the `x-bc-customer-id` header.
+Use a customer impersonation token and store it securely on your server like other secrets. When you need to run requests in the context of a particular customer (for example, if they've logged in to your application), send their BigCommerce Customer ID along with the request as the `X-Bc-customer-id` header. You could also use a regular storefront token with a customer access token. You must assign the customer access token to `X-Bc-Customer-Access-Token` in the header.
```

Comment:
I would flip this so it sounds like CAT is the recommended option and CIT is another option, but not the recommendation.